### PR TITLE
[14.0][ADD] sale_single_attribute_value_display

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
By default attributes with single value are excluded from display name of a product.
This module will include it only for sale order line description.

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
functional ok!

---
A product with this setup 

image

appears as follows in SO line description: 

image

instead of 

image

**Comment by francesco-ooops** [comment]
@pedrobaeza got an opinion on this one?

File: sale_single_attribute_value_display/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Configuration not required.
```
Comment (pedrobaeza) [inline sale_single_attribute_value_display/readme/CONFIGURE.rst]: Don't add this file if no configuration.

File: sale_single_attribute_value_display/models/models.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from odoo import models
```
Comment (pedrobaeza) [inline sale_single_attribute_value_display/models/models.py]: Call the file `product.py` instead.

File: sale_single_attribute_value_display/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Install module. Create product with single value attribute. Add it to Sale Order Line.
```
Comment (pedrobaeza) [inline sale_single_attribute_value_display/readme/USAGE.rst]: Put it as numbered bullet list:

```suggestion
#. Create a product with any single value attribute.
#. Add it to a sales order.
#. ...
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
For the module, I would choose `sale_order_line_description_single_attribute`

And a test with a `Form` of the sales order and checking the contents of the description would be very desirable.

**Comment by ilyasProgrammer** [comment]
updated

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**